microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -43,7 +43,7 @@ When you select the Address bar, a signal is sent to your default search provide
 
 You can control whether typed characters are sent to your default search provider to provide search and site suggestions.  To enable or disable sending your typed characters in the Address bar to your default search provider:
 
-1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**.<!--edge://settings/privacy-->
+1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**.<!-- todo: update as separate groups of settings --><!--edge://settings/privacy-->
```

Comment:
did the toggle **Show me search and site suggestions using my typed characters** end up in "Privacy and security" or "Search and services"?
@PatrizioBytes

Reply:
pls help integrate these incoming steps from the rewrite of the Address Bar h2 section that's in "main" branch, into this PR's scenario of separating the "Privacy, search, and services" settings into "Privacy and security" & "Search and services".